index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<!-- <script src="https://kit.fontawesome.com/b669f5742f.js" crossorigin="anonymous"></script> -->
+<link rel="stylesheet" href="./css/style.css">
+<title>Document</title>
+</head>
+<body>
+   <header id="header">
+      <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="">
+      <nav id="nav-bar">
+      <ul>
+         <li ><a class="nav-link" href="#features">Features</a></li>
+         <li ><a class="nav-link" href="#how-it-works">How It Works</a></li>
+         <li ><a class="nav-link" href="#pricing">Pricing</a></li>
+      </ul>
+      </nav>
+      </header>
+      <form id="form"  action="https://www.freecodecamp.com/email-submit">
+      <h2>Handcrafted, home-made masterpieces</h2>
+      <input name="email" id="email" type="email" placeholder="Enter your email address">
+      <input id="submit"  type="submit" value="Get Started" >
+      </form>
+      <section id="features">
+      <div class="element">
+         <!-- <i class="fa fa-3x fa-fire"> -->
+         <div class="description">
+         <h2>Premium Materials</h2>
+         <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
+      </div>
+      </div>
+
+      <div class="element">
+      <!-- <i class="fa fa-3x fa-truck"> -->
+      <div class="description">
```

Comment:
Please, do not leave unused code as comment in your codebase. Unused code as comment makes your code not clear.